source/Sashimi/CertificateEncoder.cs

```diff
@@ -141,7 +141,7 @@ static void GrantCurrentUserAccessToPrivateKeyDirectory(string privateKeyPath)
             var folderPath = Path.GetDirectoryName(privateKeyPath);
             if (folderPath == null)
                 throw new Exception("There was no directory specified in the private key path.");
-#pragma warning disable PC001
+#pragma warning disable CA1416
```

Comment:
When upgrading to net6.0 the following code fails to compile due to treatWarningsAsErrors with CA1416 warnings, (only supported on 'windows)'. This is a language feature that was introduced with net5.0. A past [example PR](https://github.com/OctopusDeploy/Shellfish/commit/6ef5cd85af52710fa36433cf0aa30bc2cc27c438) of this pragma warning being added.

In the past the default assumption was that code would be [compatible on all platforms](https://docs.microsoft.com/en-us/dotnet/core/compatibility/code-analysis/5.0/ca1416-platform-compatibility-analyzer), this hasn't changed for these libraries with the release of net6.0.

![image (1)](https://user-images.githubusercontent.com/101079287/184048078-bd6bc556-bb82-48b6-be3a-3dfc3deb10ff.png)